components/people/PeopleRow.tsx

```diff
@@ -30,13 +37,24 @@ export default function PeopleRow({
   office,
   imageURL,
 }: PeopleRowProps) {
-  const professorLink = getPath(faculty);
-  const staffLink = getPath(staff);
-  const labLink = getPath(researchLabs);
+  let href: string;
+  switch (type) {
+    case 'EMIRITUS FACULTY':
+      href = emeritusFacultyPath;
+      break;
+    case 'FACULTY':
+      href = facultyPath;
+      break;
+    case 'STAFF':
+      href = staffPath;
+      break;
+  }
+  href += `/${id}`;
+
```

Comment:
```suggestion
  const hrefList = {
  'EMIRITUS FACULTY': emeritusFacultyPath,
  'FACULTY': facultyPath,
  'STAFF': staffPath
};

const href = `${hrefList[type]}/${id}`;
```

이렇게 바꾸는게 가독성이 좋아보였습니다.